src/components/templates/Poster/Poster.tsx

```diff
@@ -0,0 +1,169 @@
+            <p className="poster__title">{venue.title}</p>
+            <div className="poster__categories" />
+          </div>
+
+          <div className="poster__header--right-cell">
+            {isMeActiveParticipant && (
+              <div
+                className="poster__control poster__control--stop"
+                onClick={turnVideoOff}
+              >
+                <FontAwesomeIcon icon={faStop} size="lg" />
+                <span className="poster__control-text">Stop video</span>
+              </div>
+            )}
+
+            <OverlayTrigger
+              trigger="click"
+              placement="bottom-end"
+              overlay={
+                <Popover
+                  id="popover-basic"
+                  className="poster__settings-popover"
+                >
+                  <Popover.Content className="poster__settings-popover-content">
+                    <Toggler
+                      type="checkbox"
+                      className="switch-hidden-input"
+                      checked={!!isPosterLive}
+                      onChange={isPosterLive ? setVenueLiveOff : setVenueLiveOn}
+                      title={
+                        isPosterLive ? "Poster is live" : "Make poster live"
+                      }
+                    />
+                  </Popover.Content>
+                </Popover>
+              }
+              rootClose={true}
+            >
+              <div className="poster__control">
+                <FontAwesomeIcon icon={faCog} size="lg" />
+                <span className="poster__control-text">Settings</span>
+              </div>
+            </OverlayTrigger>
+
+            <div className="poster__control">
+              <FontAwesomeIcon icon={faShare} size="lg" />
+              <span className="poster__control-text">Share</span>
+            </div>
+
+            {venue.introVideoUrl && (
+              <div className="poster__control" onClick={showIntroVideoModal}>
+                <FontAwesomeIcon icon={faTv} size="lg" />
+                <span className="poster__control-text">Intro Video</span>
+              </div>
+            )}
+          </div>
+        </div>
+
+        <div className="poster__content">
+          <iframe
+            className="poster__iframe"
+            src={venue.iframeUrl}
+            title="poster-iframe"
+          />
+
+          {videoParticipants}
+
+          {hasFreeSpace && !isMeActiveParticipant && (
+            <div
+              className="poster__join-video-participants-btn"
+              onClick={turnVideoOn}
+            >
+              Join with video
+            </div>
+          )}
+        </div>
+
+        <div className="poster__listeners">
+          <UserList users={passiveListerens} activity="listening" />
+        </div>
+      </div>
+      <IntroVideoPreviewModal
+        isVisible={isIntroVideoShown}
+        onHide={hideIntroVideoModal}
+        posterVenue={venue}
+      />
+    </>
```

Comment:
I would probably just include the modal within the main container div, and then get rid of the fragment:

```suggestion
        <div className="poster__listeners">
          <UserList users={passiveListerens} activity="listening" />
        </div>

        <IntroVideoPreviewModal
          isVisible={isIntroVideoShown}
          onHide={hideIntroVideoModal}
          posterVenue={venue}
        />
      </div>
```